### DISCUSSION TITLE
is it possible deploy the web application at the root of a Tomcat?

### DISCUSSION THREAD

**Discussion by ricva84** [SCADA-LTS/Scada-LTS#2536] [Q&A]
If this is not the space for this query, please excuse me.
Thank you for the work and time dedicated to the project.

Is it possible that instead of loading at example.com/Scada-LTS, you can just use example.com?

In docker, when I rename Scada-LTS.war to ROOT.war, I get errors on the site.
As in the Script tab, it doesn't finish loading or alert popups in all tabs.

**Comment by hjbleier** [comment]
I have tried the same with no success either, it appears that there are lots of things that are hard-coded to look for the path /Scada-LTS/ in the project unfortunately. But I am no means a expert in Apache tomcat deployments, just have tried myself tinkering around with what you are attempting in the past. Hopefully someone will chime in a give a solution to do this, because I am also very interested in doing this, it would be very nice to shorten up the URL to access my Scada-LTS deployments. I would like to do something like https://Scada-LTS.domainname.com or https://scadalts.domainname.com that would be much nicer, currently I have it at https://scadalts.domainname.com/Scada-LTS/ to make it work (I got lots of other subdomains, https://gitlab.domainname.com, etc.).

**Comment by Limraj** [comment]
Hi @ricva84 and @hjbleier,
Before starting tomcat, theoretically it is enough to change the name of the war file from Scada-LTS.war to ROOT.war and run tomcat, but in fact something may not work completely, if necessary, we can check and correct it so that it works without Scada- LTS.

Regards,
Kamil Jarmusik

**Comment by Limraj** [comment]
Created issue: https://github.com/SCADA-LTS/Scada-LTS/issues/2864

@ricva84 and @hjbleier 
Let me know if I still need this configuration.

**Comment by Limraj** [comment]
From version 2.7.8 is possible: https://github.com/SCADA-LTS/Scada-LTS/releases/tag/v2.7.8 .

**Comment by rafaelpsousa** [comment]
I tested it on docker with the version 2.7.8 and it works
